app/enterprise/0.32-x/upgrades-migrations.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Upgrades and Migrations
+---
+
+# Upgrades and Migrations
+## Upgrades of Kong EE
+
+We define an upgrade, the process of changing the version of the software currently deployed to a more recent one, there is no difference with concept of update; normally associated with an upgrade between minor versions.
+
+We suggest to follow the below steps to perform an effective upgrade, this approach will require downtime, in case you need high availability review the Blue/Green Deployment section.
+
+1. Download the destination version of the product you are interest to upgrade to in Bintray
```

Comment:
`1. Download the version you are interested in upgrading to`?

Reply:
Please avoid mentioning Bintray - instead, you can link the word "download" to our Bintray URL.

Reply:
"...of the product..." should be specified to something like "...Kong..."

Reply:
Bintray is already mentioned in EE docs here: https://docs.konghq.com/enterprise/0.32-x/installation/docker/, shall we keep consistency doing a deep screening of where Bintray is mentioned in EE docs?

Reply:
Yep, eventually we should. In the case of this PR, we should at least make this PR like how we want our other docs to be (rather than leaving this PR to get "fixed" later)